.github/workflows/main.yml

```diff
@@ -77,12 +77,20 @@ jobs:
             publish: true
 
           - name: windows-python3
-            os: windows-2019
+            os: windows-2016
             buildType: RELEASE
             options: .github/workflows/main/options.windows
             dependenciesURL: https://github.com/hypothetical-inc/gafferDependencies/releases/download/3.1.1/gafferDependencies-3.1.1-Python3-windows.zip
             tests: testCore testCorePython testScene testImage testAlembic testVDB
             publish: false
+          
+          - name: windows-python3-debug
```

Comment:
This one is showing up with a very verbose name in the results :

![image](https://user-images.githubusercontent.com/1133871/148376757-13128212-33f5-427f-b696-e6967a9599c2.png)

I think that might be fixed by adding `windows-python3-debug` to the list of names on L37.

Reply:
Fixed in https://github.com/ImageEngine/cortex/pull/1217/commits/9bd776bccfa77b8734910c1a120cd7404c8cf2ba